gnd/src/main/java/com/google/android/gnd/persistence/local/LocalDataStore.java

```diff
@@ -145,4 +145,6 @@
    * the area into the local data store.
    */
   Completable insertOrUpdateOfflineArea(OfflineArea area);
+
+  Flowable<ImmutableList<OfflineArea>> getOfflineAreas();
```

Comment:
As discussed on hangouts, we should call this `getOfflineAreasOnceAndStream`

Reply:
Totally forgot, thanks! 6724e83